src/main/java/com/epam/spring/time_tracking/repository/CategoryRepo.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.spring.time_tracking.repository;
+
+import com.epam.spring.time_tracking.model.Category;
+
+import java.util.List;
+
+public interface CategoryRepo {
+
+    List<Category> getCategories();
+
+    Category getCategory(int categoryId);
```

Comment:
Total nitpick, but I would use something like getCategoryById(), we have better understanding by which field we are trying to get the category. Also, for instance in future you will have other options to find category, by name.